projects/shared/src/lib/form/protocol-field-set/protocol-field-set.component.html

```diff
@@ -0,0 +1,92 @@
+<div class="form-group" [formGroup]="form">
+  <label i18n="protocol label main@@protocolLabelMain">Harvest protocol *</label>
+  <div class="form-group-radios">
+    <label *ngIf="hasFileOption" i18n="protocol radio file@@protocolRadioFile">
+      <input type="radio" [formControlName]="protocolSwitchField" value="FILE" />
+      File upload
+    </label>
+    <label i18n="protocol radio oai-pmh@@protocolRadioOaiPmh">
+      <input type="radio" [formControlName]="protocolSwitchField" value="OAIPMH_HARVEST" />
+      OAI-PMH upload
+    </label>
+    <label i18n="protocol radio http@@protocolRadioHttp">
+      <input type="radio" [formControlName]="protocolSwitchField" value="HTTP_HARVEST" />
+      HTTP upload
+    </label>
```

Comment:
For these, shouldn't we see the selection upon opening the page? If I load an existing dataset and select the harvesting step both radios are unselected.

Reply:
They do show if the step is enabled, they don't show if it isn't, but that's an existing bug that I think will require back-end changes to fix.